otaclient/app/create_standby/rebuild_mode.py

```diff
@@ -112,21 +113,24 @@ def _process_regulars(self):
         logger.info("start applying delta...")
         with ThreadPoolExecutor(thread_name_prefix="create_standby_slot") as pool:
             _mapper = RetryTaskMap(
-                self._process_regular,
-                self.delta_bundle.new_delta.items(),
                 title="process_regulars",
                 max_concurrent=cfg.MAX_CONCURRENT_PROCESS_FILE_TASKS,
-                backoff_max=cfg.CREATE_STANDBY_BACKOFF_MAX,
-                backoff_factor=cfg.CREATE_STANDBY_BACKOFF_FACTOR,
                 max_retry=cfg.CREATE_STANDBY_RETRY_MAX,
+                retry_interval_f=partial(
+                    wait_with_backoff,
+                    _backoff_factor=cfg.CREATE_STANDBY_BACKOFF_FACTOR,
+                    _backoff_max=cfg.CREATE_STANDBY_BACKOFF_MAX,
+                ),
                 executor=pool,
             )
-            for _exp, _entry, _ in _mapper.execute():
-                if _exp:
+            for _is_successful, _entry, _fut in _mapper.map(
+                self._process_regular,
+                self.delta_bundle.new_delta.items(),
+            ):
+                if not _is_successful:
                     logger.error(
-                        f"[process_regular] failed to process {_entry=}: {_exp=}"
+                        f"[process_regular] failed to process {_entry=}: {_fut=}"
                     )
-                    raise _exp
```

Comment:
This raise is not intended, we only break out when max_retry is reached, and let the RetryTaskMap instance raise the last exception.